R/check_lintr.R

```diff
@@ -1,5 +1,24 @@
 #' Check the packages for linters
 #'
+#' This functions does [static code analysis](
+#' https://en.wikipedia.org/wiki/Static_program_analysis).
+#' It relies on [lintr::lint_package()].
+#' We recommend that you activate all code diagnostics in RStudio.
+#' You can find this in the menu _Tools_ > _Global options_ > _Code_ >
+#' _Diagnostics_.
+#'
+#' @details
+#'
+#' Your code must follow the default coding style defined by the lintr package.
+#' - Use underscore (`_`) to separate long names.
+#' - No names longer than 30 characters.
+#' - No lines longer than 80 characters.
+#' - Use `TRUE` and `FALSE`,
+#'   Don't use `T` or `F`.
+#' - Add sufficient whitespace characters.
+#'   But don't use whitespace when it is not relevant (end of line or file).
+#' - ...
```

Comment:
This list is certainly clear enough to grasp the idea, but for those who are not familiar with the package, it could be a nice-to-have to add a link to the complete list of linters in the package. (github.com/jimhester/lintr#available-linters)